weblogic-azure-aks/src/main/arm/scripts/buildWLSDockerImage.sh

```diff
@@ -158,35 +172,15 @@ function prepare_wls_models() {
 CLUSTER_SIZE=${wlsClusterSize}
 EOF
 
-    # Generate application deployment model in model.yaml
-    # Known issue: no support for package name that has comma.
-    # remove []
-    if [ "${appPackageUrls}" == "[]" ]; then
-        return
-    fi
+    echo "Starting generating image model file..."
```

Comment:
Starting generation of image model file...

Reply:
Fixed in https://github.com/galiacheng/weblogic-azure/commit/e599233edb163bdfa597ea2551aee0771f364913